ui/admin/app/components/credential-stores/credential-store/header/edit-worker-filter.hbs

```diff
@@ -0,0 +1,11 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<h2>
+  {{t 'resources.credential-store.worker-filter.messages.edit.title'}}
+  <DocLink @doc='worker-filters-format' @iconSize='large' />
+</h2>
+<p>{{t 'resources.credential-store.worker-filter.messages.edit.description'}}
+</p>
```

Comment:
👋 Yep I chatted with @lisbet-alvarez about this and she said since the Hds migration is a separate endeavor, she suggested I follow existing conventions for now. Happy to do whatever makes sense for this feature. Lmk!

Reply:
I think we've been just using HDS for anything new, it's not blocking as it's simple either way

Reply:
I agreed with Zed.

For text and for all new elements, except specific cases, we should be using HDS.